ush/get_extrn_mdl_file_dir_info.sh

```diff
@@ -460,7 +460,7 @@ and analysis or forecast (anl_or_fcst):
       fcst_hh=( $( printf "%02d " "${lbc_spec_fhrs[@]}" ) )
 
       if [ "${MACHINE}" = "JET" ]; then 
-        prefix="wrfnat_130_"
```

Comment:
I think these lines might cause some issues for CONUS. Can we make use of the PREFIX env variable you added in set_extrn_mdl_params or something similar?

Reply:
Sure, I can use that instead since it's available.  Note that switching from 130 to rr grids won't cause any issues for CONUS other than adding a small amount of processing time to make_ics and make_lbcs.  I'd argue it's a bit better to use these native grids anyway, since rotated lat-lon to Lambert interpolation is involved in creating the 130 grids

Reply:
Great! Thanks. If they can be used for both, no reason to use the PREFIX variable, I think.